### PR TITLE
Modernizing Winforms images in core

### PR DESCRIPTION
**Pull request by dreddy-work**
Current images in winforms run-time are Bitmaps that are out of synch  in 'style' with Visual Studio. ALso, bitmaps are hard to scale  on high resolution screens without loosing quality ( results in blurry images). As part of this change we are replacing those images with Icons that can match with Visual Studio styling and can be extended to support up to 300% DPI scaling in the future.  These Icons may differ in the way they look from previous images but are in synch with what  Visual Studio uses.

### CODE REVIEW

**Comment by Tanya-Solyanik** [comment]
I'm surprised that the .csproj file is not changed.

**Comment by zsd4yr** [comment]
Looks like some bitmap-specific unit tests have failed. I suggest refactoring them to perform the appropriate behavior for icon files. Also this is a compile-time change people could use to see that these are icons now, correct?

**Comment by zsd4yr** [comment]
> I'm surprised that the .csproj file is not changed.

It was though.

**Comment by dreddy-work** [comment]
> Looks like some bitmap-specific unit tests have failed. I suggest refactoring them to perform the appropriate behavior for icon files. Also this is a compile-time change people could use to see that these are icons now, correct?

They failed because they are bu default transparent and we were explicitly calling transparent by assuming they were bitmaps. I have changes the source code ( not tests)  to expect Icons.

**Comment by zsd4yr** [comment]
@dreddy-work is this ready?

**Comment by dreddy-work** [comment]
This is now ready. @zsd4yr

File: src/Common/src/DpiHelper.cs [hunk 1]
```diff
@@ -292,6 +292,17 @@ public static Bitmap CreateResizedBitmap(Bitmap logicalImage, Size targetImageSi
             return ScaleBitmapToSize(logicalImage, targetImageSize);
         }
 
+        /// <summary>
+        /// Creating bitmap from Icon resource
+        /// </summary>
+        public static Bitmap GetBitMapFromIcon(Type t, string name)
```
Comment (weltkante) [inline src/Common/src/DpiHelper.cs]: just a side note, `Bitmap` in WinForms/System.Drawing is used as one word not `BitMap` being two words

Comment (dreddy-work) [reply src/Common/src/DpiHelper.cs]: Agree. Making the change.

**Comment by dreddy-work** [comment]
@Tanya-Solyanik , cna you please take a look and sign off, corresponding corefx change is now merged. this PR is validated and ready to merge

**Comment by dreddy-work** [comment]
See #886

**Pull request closed without merging**